### DISCUSSION TITLE
Hook into before form schema

### DISCUSSION THREAD

**Discussion by pajkho** [filamentphp/filament#9708] [Ideas]
We have a form field that we want in 90% of all forms. Instead of manually having to add this to all form schemas I would like to find an automatic way to do this based on a certain condition. What would be the best way to achieve this?

Have tried by extending classes etc but can't find a good solution to hook into the form schema and add fields in a parent class or so. Is there any other way we can hook in at the beginning or end of form schema? Or any other good solution that is SOLID?

Any help is highly appreciated. Or is this possibly a new feature that needs to be added?

**Comment by twltwk** [comment]
I'll raise the subject again. Would it be possible to find out more ?

**Reply by pajkho** [reply]
Sure, let me know what more info you would need.
